src/main/javassist/bytecode/analysis/MultiArrayType.java

```diff
@@ -108,16 +108,10 @@ public boolean isAssignableTo(Type type) {
             return false;
 
         if (typeDims < dims) {
-            if (eq(typeRoot.getCtClass(), Type.OBJECT.getCtClass()))
-                return true;
-
-            if (eq(typeRoot.getCtClass(), Type.CLONEABLE.getCtClass()))
-                return true;
-
-            if (eq(typeRoot.getCtClass(), Type.SERIALIZABLE.getCtClass()))
-                return true;
-
-            return false;
+            CtClass c = typeRoot.getCtClass();
+            return eq(c, Type.OBJECT.getCtClass()) 
+                    || eq(c, Type.CLONEABLE.getCtClass()) 
+                    || eq(c, Type.SERIALIZABLE.getCtClass());
```

Comment:
Avoids calling getCtClass() multiple times (performance) and reduces the code size.